### DISCUSSION TITLE
Pro benefits not available after renewing Student Developer Pack

### DISCUSSION THREAD

**Discussion by crizmo** [community/community#69430] [GitHub Education] [answered]
### Hi everyone,

I renewed my Student Developer Pack by submitting my college ID card. 
The application got approved, but I'm still being asked to pay for GitHub Pro. How do I fix this ?

![image](https://github.com/community/community/assets/83665497/f10de920-56f0-459a-801c-75ab38db7742)
![image](https://github.com/community/community/assets/83665497/71d9798f-c5d9-4750-a16b-aa735a9cab4d)

**Comment by salimovrashad** [comment]
Response #10285

**Reply by crizmo** [reply]
I need to have 2fa to get access to the benefits , is that what you mean ?

**Comment by mantasu** [comment, marked as answer]
I think you're having the same issue as some other people. See the answer [here](https://github.com/orgs/community/discussions/69264)

Basically, you may still need to wait for a few more days as it seems the staff is super busy currently. It doesn't hurt to open a [ticket](https://support.github.com/request/education), just remember to close it once your benefits arrive.

**Reply by crizmo** [reply]
thanks for replying !
i made a ticket about the same and got this reply

![image](https://github.com/community/community/assets/83665497/cc95c7f3-1660-4f4f-b457-71eb2f78d01d)

but idk where the help available tile is and does it redirect to the support itself ?

**Reply by muhammedfurkan** [reply]
same here

**Reply by mantasu** [reply]
If it's about trying to access Github Copilot, did you try clearing the browser cache? Maybe a similar answer [here](https://github.com/orgs/community/discussions/39703#discussioncomment-4256521) can also help?

**Reply by crizmo** [reply]
didnt work

**Reply by mantasu** [reply]
Might be worth a shot to contact the Copilot maintainer directly [here](https://copilot-language.github.io/contact.html).

**Reply by crizmo** [reply]
thats fine but , its more about getting the benefits as a whole than just for copilot for me atleast
plus it doesnt let me resubmit the benefits form thingy - says i have already applied and got the student benefits

**Reply by mantasu** [reply]
You could try opening a new ticket (and attach some screenshots too) but let them know that you cannot access the benefits even though you are approved.

**Reply by crizmo** [reply]
i think i'll wait for some days , if i dont get any reply till then , then i might make a new ticket
thanks for all the help btw

**Reply by mantasu** [reply]
No worries, I hope they can solve it for you soon!

**Comment by crizmo** [comment]
![image](https://github.com/community/community/assets/83665497/8184ba42-0d79-4465-ba8e-e3d62e010266)

Got a reply on the ticket and got the benefits !
Writing this for people facing similar issues

Open a ticket on GitHub's support site if you don't get the benefits in desired time 
Otherwise GitHub them self will reply on mail about your benefits weather you should re-verify or not

Thank you all for helping !
